qtp_diversity/tests/test_summary.py

```diff
@@ -139,7 +139,7 @@ def _create_job(self, a_type, files):
                 'name': "A name", 'analysis': 1, 'data_type': '16S'}
         aid = self.qclient.post('/apitest/artifact/', data=data)['artifact']
         parameters = {'input_data': aid}
-        data = {'command': dumps(['Diversity types', '0.1.1',
+        data = {'command': dumps(['Diversity types', '2021.05',
```

Comment:
Or maybe even 2021.03? qtp sequencing has 2021.03.
```suggestion
        data = {'command': dumps(['Diversity types', '2021.04',
```

Reply:
Good question, I have been adding the date when it will be deployed in Qiita for all Qiita plugin, except Qiime2 - that one has the actual qiime2 deployment date to avoid confusions.